### ISSUE TITLE
Contentstack integration

### ISSUE THREAD

**Issue by Fedeorlandau** [valtech-ch/ring#95]
I would love to implement the Contentstack integration, @lorenzogm are you open to it? 

Thanks!

**Comment by lorenzogm** [comment]
Absolutely, the main requirement is to follow the contracts (libs/contracts). Please let me know if you need some support
